android-project/app/build.gradle

```diff
@@ -29,6 +29,10 @@ android {
 			proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
 		}
 	}
+	applicationVariants.all { variant ->
+		tasks["merge${variant.name.capitalize()}Assets"]
+			.dependsOn(externalNativeBuildDebug)

```

Comment:
Actually.. I wonder if this needs to be `externalNativeBuild${variant.name}` or something...